lib/src/core/models/functions/functions.dart

```diff
@@ -0,0 +1,120 @@
+import 'dart:convert';
+
+import 'package:meta/meta.dart';
+
+@immutable
+class OpenAiFunctionModel {
```

Comment:
Can we rename this class to `OpenAIFunctionModel` with an `AI` instead of `Ai`

Reply:
Done

Reply:
Thank you